README.rst

```diff
@@ -15,18 +15,29 @@ Building
 
 To build the docs locally, first install ``sphinx``, ``furo`` and
 ``sphinx-copybutton``.
-On Fedora this can be with::
+
+On **Fedora** this can be done with::
 
   sudo dnf install python3-sphinx python3-furo python-sphinx-copybutton python-sphinxext-opengraph
 
-On Debian this can be with::
+On **Debian** this can be done with::
 
   sudo apt install python3-sphinx python3-furo python-sphinx-copybutton python-sphinxext-opengraph
 
-Then run ``make html`` in the ``docs`` directory.
+With **pip**, this can be done with::
+
+  pip install -r requirements.txt
+
+Then, build the documentation::
+
+  cd docs
+  make html
```

Comment:
Change this to `make -C docs html`

Reply:
Also add the linkcheck command that's run on CI

Reply:
Done in 27becf6